Source/WebCore/PAL/pal/spi/cocoa/AppSSOSPI.h

```diff
@@ -33,6 +33,12 @@ DECLARE_SYSTEM_HEADER
 
 #import <AppSSO/AppSSO.h>
 #import <AppSSOCore/AppSSOCore.h>
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SOAuthorization (Staging)
```

Comment:
Usually I see Staging_rdarXXX so we know when it can be removed. Also spacing seems weird above.